requirements_all.txt

```diff
@@ -708,6 +708,13 @@ gitterpy==0.1.7
 
 # homeassistant.components.glances
 glances_api==0.3.4
+glances_api==0.2.0
+
+# homeassistant.components.glinet
+gli_py==0.0.12
+
+# homeassistant.components.gntp
+gntp==1.0.3
```

Comment:
I think you can only add requirements for your own component

Reply:
accidental leftovers from a rebase